config/config-template-long.yaml

```diff
@@ -133,5 +143,26 @@ scanners:
       # If set to True and authentication is oauth2_rtoken and api.apiUrl is set, download the API outside of ZAP
       oauth2OpenapiManualDownload: False
 
+    defectDojoExport:
+      type: "reimport" # choose betwee: import, reimport, False (disable export). Default (or other content): re-import if test is set
+      # Parameters contain data that will directly be sent as parameters to DefectDojo's import/reimport endpoints.
+      # For example: commit tag, version, push_to_jira, etc.
+      # See https://demo.defectdojo.org/api/v2/doc/ for a list of possibilities
+      # The minimum set of data is whatever is needed to identify which engagement/test needs to be chosen.
+      # If neither a test ID (`test` parameter), nor product_name and engagement_name were provided, sane default will be attempted:
+      #   - product_name chosen from either application.productName or application.shortName
+      #   - engagement_name:  "RapiDAST" [this way the same engagement will always be chosen, regardless of the scanner]
+      parameters:
+        product_name: "My Product"
+        engagement_name: "RapiDAST"
+          # - or -
+        engagement: 3   # engagement ID
```

Comment:
since only one of the options is used with the 'or' comment, what do you think about commenting out others leaving one option?